src/CMakeLists.txt

```diff
@@ -58,6 +58,14 @@ if(UMF_BUILD_OS_MEMORY_PROVIDER)
     endif()
 endif()
 
+if(UMF_BUILD_ZE_MEMORY_PROVIDER)
+    set(UMF_SOURCES_LINUX ${UMF_SOURCES_LINUX}
+        provider/provider_ze_memory.c)
+    if(LINUX)
+        set(UMF_LIBS ${UMF_LIBS} ze_loader)
```

Comment:
We should either:
- look for L0 using find_package() or similar
- use FetchContent to download L0 header (UR does that)

Reply:
I changed this and ze_loader is loaded in runtime using dlopen